src/components/Footer.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import classNames from 'classnames';
+import { FilterValues } from '../constants';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  onChange: React.Dispatch<React.SetStateAction<FilterValues>>;
+  selectedFilter: string;
+  clearCompletedTodos: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  onChange,
+  selectedFilter,
+  clearCompletedTodos,
+}) => {
+  const notCompletedTodos = [...todos].filter(todo => !todo.completed);
+  const todosCountMessage = `${notCompletedTodos.length} items left`;
+  const isClearCompletedDisabled = todos.length === notCompletedTodos.length;
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">{todosCountMessage}</span>
+
+      <nav className="filter">
+        { Object.values(FilterValues).map((value) => (
+          <a
+            href="#/"
+            key={value}
+            className={classNames('filter__link', {
+              selected: selectedFilter === value,
+            })}
+            onClick={() => onChange(value)}
+          >
+            {value}
+          </a>
+        )) }
```

Comment:
cool solution👍 Just do not use anchor tag, it's useless here, but it's ok as it was in the task by default, so it's up to you to change it

Reply:
if I change it, footer doesn't work as I expect